Packs/Elasticsearch/ReleaseNotes/1_3_15.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Elasticsearch v2
+- Updated the Docker image to: *demisto/py3-tools:1.0.0.72003*.
+- Updated the default datetime format to YYYY-MM-DDTHH:mm:ss.SSSZ. (Elasticsearch standard).
+- Fixed an issue where an index regex was set in the index parameter (it will now take the custom format of the first index).
```

Comment:
```suggestion
- Fixed an issue where an index regex was set in the index parameter, from now on it will take the custom format from the first index.
```